Gemfile

```diff
@@ -14,6 +14,7 @@ gem "rake", ">= 11.1"
 gem "mocha", require: false
 
 gem "capybara", ">= 2.15", "< 4.0"
+gem "selenium-webdriver", "~> 3.5.1"
```

Comment:
Is there a way to test this without adding selenium-webdriver to Rails itself? If this is for applications it should be added to the generated Gemfile instead.

Reply:
selenium-wedbdriver is currently a sub-dependency of a couple gem ([blade_sauce_plugin](https://github.com/rails/rails/blob/f815e773412f1064492507d76a9a011e6dde6042/Gemfile#L83) and [qunit](https://github.com/rails/rails/blob/f815e773412f1064492507d76a9a011e6dde6042/Gemfile#L97))

I just made it an explicit dependency since we are directly using it now.